go/chat/tlf.go

```diff
@@ -133,6 +166,75 @@ func (t *KBFSTLFInfoSource) CompleteAndCanonicalizePrivateTlfName(ctx context.Co
 	return resp.NameIDBreaks, nil
 }
 
+func (t *KBFSTLFInfoSource) identifyTLF(ctx context.Context, arg keybase1.TLFQuery, private bool) ([]keybase1.TLFIdentifyFailure, error) {
+	var fails []keybase1.TLFIdentifyFailure
+	pieces := strings.Split(strings.Fields(arg.TlfName)[0], ",")
+	for _, p := range pieces {
+		f, err := t.identifyUser(ctx, p, private, arg.IdentifyBehavior)
+		if err != nil {
+			return nil, err
+		}
+		fails = append(fails, f)
+	}
+	return fails, nil
+}
+
+func (t *KBFSTLFInfoSource) identifyUser(ctx context.Context, assertion string, private bool, idBehavior keybase1.TLFIdentifyBehavior) (keybase1.TLFIdentifyFailure, error) {
+
+	// if an identify UI is registered, then use it
+	sessionID, idUI, err := t.G().UIRouter.GetIdentifyUICtx(ctx)
+	if err != nil {
+		return keybase1.TLFIdentifyFailure{}, err
+	}
+
+	if idUI == nil {
+		if t.identifier != nil {
+			t.G().Log.Debug("using KBFSTLFInfoSource.identifier to identify")
+			return t.identifier.Identify(ctx, assertion, private, idBehavior)
+		}
+
+		sessionID = 0
+		idUI = chatNullIdentifyUI{}
+	}
+
+	reason := "You accessed a public conversation."
```

Comment:
Since we never show any tracker popups for chat, can we just pass the null guy everytime?